src/MarginTrading.Backend.Services/VolumeEquivalentService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using MarginTrading.Backend.Core;
+using MarginTrading.Backend.Core.Settings;
+
+namespace MarginTrading.Backend.Services
+{
+	public class VolumeEquivalentService : IVolumeEquivalentService
```

Comment:
I don't quite understand why this has "volume" in it's name, despite that it calculates equivalent prices.
I suggest something like `EquivalentPricesService` - this order of words is more natural.

Reply:
Fixed